# left mbot arm position controller parameters

left_arm_joint0_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_arm_joint0
  pid: {p: 50.0, i: 0.01, d: 5.0}

left_arm_joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_arm_joint1
  pid: {p: 50.0, i: 0.01, d: 5.0}

left_arm_joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_arm_joint2
  pid: {p: 50.0, i: 0.01, d: 5.0}

left_arm_joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_arm_joint3
  pid: {p: 50.0, i: 0.01, d: 5.0}

left_arm_joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_arm_joint4
  pid: {p: 50.0, i: 0.01, d: 5.0}

left_arm_joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_arm_joint5
  pid: {p: 50.0, i: 0.01, d: 5.0}

left_arm_joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: left_arm_joint6
  pid: {p: 50.0, i: 0.01, d: 5.0}

